src/client/pydaos/raw/daos_api.py

```diff
@@ -6,6 +6,9 @@
 """
 # pylint: disable=pylint-too-many-lines
 
+# pylint: disable=import-error
+from .. import pydaos_shim as pydaos_shim
```

Comment:
(pylint-relative-beyond-top-level) Attempted relative import beyond top-level package
(pylint-useless-import-alias) Import alias does not rename original package